freqtrade/rpc/telegram.py

```diff
@@ -74,10 +76,48 @@ def __init__(self, freqtrade) -> None:
 
         self._updater: Updater
         self._config = freqtrade.config
+        self._init_keyboard()
         self._init()
         if self._config.get('fiat_display_currency', None):
             self._fiat_converter = CryptoToFiatConverter()
 
+    def _init_keyboard(self) -> None:
+        """
+        Validates the keyboard configuration from telegram config
+        section.
+        """
+        self._keyboard: List[List[Union[str, KeyboardButton]]] = [
+            ['/daily', '/profit', '/balance'],
+            ['/status', '/status table', '/performance'],
+            ['/count', '/start', '/stop', '/help']
+        ]
+        # do not allow commands with mandatory arguments and critical cmds
+        # like /forcesell and /forcebuy
+        # TODO: DRY! - its not good to list all valid cmds here. But otherwise
+        #       this needs refacoring of the whole telegram module (same
+        #       problem in _help()).
+        valid_keys: List[str] = ['/start', '/stop', '/status', '/status table',
+                                 '/trades', '/profit', '/performance', '/daily',
+                                 '/stats', '/count', '/locks', '/balance',
+                                 '/stopbuy', '/reload_config', '/show_config',
+                                 '/logs', '/whitelist', '/blacklist', '/edge',
+                                 '/help', '/version']
+
+        # custom keyboard specified in config.json
+        cust_keyboard = self._config['telegram'].get('keyboard', [])
+        if cust_keyboard:
+            # check for valid shortcuts
+            invalid_keys = [b for b in chain.from_iterable(cust_keyboard)
+                            if b not in valid_keys]
+            if len(invalid_keys):
+                err_msg = ('config.telegram.keyboard: invalid commands for '
+                           f'custom keyboard: {invalid_keys}')
```

Comment:
maybe in a more ... talking way?
like `Invalid commands for custom Telegram keyboard: {invalid_keys}` ... 

or even `Invalid command for custom keyboard {invalid_keys}. \n valid keys are: {valid_keys}`  (in pseudo code though)

that way it's immediately clear what are possible values without even consulting the docs ... 

yes, the "location" will be wrong - but that's the case with most validations that raise exceptions (just look at `config_validation.py` for examples)

Reply:
done